docs/2.guide/3.going-further/7.layers.md

```diff
@@ -160,13 +160,14 @@ When importing using aliases (such as `~/` and `@/`) in a layer components and c
 Also when using relative paths in `nuxt.config` file of a layer, (with exception of nested `extends`) they are resolved relative to user's project instead of the layer. As a workaround, use full resolved paths in `nuxt.config`:
 
 ```js [nuxt.config.ts]
-import { createResolver } from '@nuxt/kit'
+import { fileURLToPath } from 'url'
+import { dirname, join } from 'path'
 
-const { resolve } = createResolver(import.meta.url)
+const __dirname = dirname(fileURLToPath(import.meta.url))
```

Comment:
Might not matter here, but I would advise avoiding `__dirname` as it's sometimes used to detect a CJS environment.

Reply:
I agree that we might encourage all ESM syntax with `impoort.meta.url`

BTW jiti probably will always support CJS (`require.resolve` is even available in `nuxt.config` and works perfectly fine...) it is only and only matter of consistency to not use this simpler syntax..

Reply:
I mean, avoid the variable name `__dirname` as a matter of good practice.

Reply:
I understand and it is a nice practice to stick with ESM syntax.

Only mentioning it _is_ a cjs environment by jiti for loading config. there should be no issues with it.